dockstore-webservice/src/main/java/io/dockstore/webservice/core/Tool.java

```diff
@@ -139,6 +139,11 @@
             + "If tool was never built on quay.io, then last build will be null. N/A for hosted/manual path tools", position = 25)
     private Date lastBuild;
 
+    @Column(nullable = false, columnDefinition = "varchar default ''")
```

Comment:
If this cannot be null but an endpoint is required to migrate, will the site still run ok after migrating the DB to include the new column but before running the migration script you wrote?

Reply:
got rid of the endpoint and replaced with an sql migration. The UI doesn't like null values so the default is an empty string for when cwl or wdl sourcefiles can't be found for a tool.